docs: (core) add bar chart list card example

dimamarksman opened the pull request:
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/3508
#### Please provide a brief summary of this pull request.
The purpose of this PR is to extend core `Card` documentation by adding `bar chart list card` example and extend `analytical card` example by adding a chart using third party library for that. 

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

InnaAtanasova commented:
1. I renamed the PR to docs as it's only touching the documentation part
2. When you first load the page the content of the Analytical card is empty (it takes a bit to load the GoogleChartService). Maybe use a condition to show the card loader until the GoogleChartService loads
3. A small styling issue - the right border of the content is hidden under the inner content:
<img width="898" alt="Screen Shot 2020-09-30 at 8 33 35 AM" src="https://user-images.githubusercontent.com/39598672/94686098-52d88a80-02f8-11eb-84fd-c14d504ebfe2.png">

dimamarksman commented:
> 1. I renamed the PR to docs as it's only touching the documentation part
> 2. When you first load the page the content of the Analytical card is empty (it takes a bit to load the GoogleChartService). Maybe use a condition to show the card loader until the GoogleChartService loads
> 3. A small styling issue - the right border of the content is hidden under the inner content:
> 
> <img alt="Screen Shot 2020-09-30 at 8 33 35 AM" width="898" src="https://user-images.githubusercontent.com/39598672/94686098-52d88a80-02f8-11eb-84fd-c14d504ebfe2.png">

@InnaAtanasova I added loader and fixed styles. Thanks!
